src/main/java/core/mate/academy/service/ExcavatorProducer.java

```diff
@@ -0,0 +1,15 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Excavator;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer<Excavator> {
+    @Override
+    public List<Excavator> get() {
+        return List.of(
+                new Excavator(),
+                new Excavator(),
+                new Excavator()

```

Comment:
use parameterized constructors. at the moment, you've added fields to this class, but not initialized them